dolphinscheduler-server/src/main/resources/config/install_config.conf

```diff
@@ -30,29 +30,29 @@ sshPort="22"
 
 # A comma separated list of machine hostname or IP would be installed Master server, it
 # must be a subset of configuration `ips`.
-# Example for hostnames: ips="ds1,ds2", Example for IP: ips="192.168.8.1,192.168.8.2"
+# Example for hostname: masters="ds1,ds2", Example for IP: masters="192.168.8.1,192.168.8.2"
 masters="ds1,ds2"
 
 # A comma separated list of machine <hostname>:<workerGroup> or <IP>:<workerGroup>.All hostname or IP must be a
 # subset of configuration `ips`, And workerGroup have default value as `default`, but we recommend you declare behind the hosts
-# Example for hostnames: ips="ds1:default,ds2:default,ds3:default", Example for IP: ips="192.168.8.1:default,192.168.8.2:default,192.168.8.3:default"
+# Example for hostname: workers="ds1:default,ds2:default,ds3:default", Example for IP: workers="192.168.8.1:default,192.168.8.2:default,192.168.8.3:default"
 workers="ds1:default,ds2:default,ds3:default,ds4:default,ds5:default"
 
 # A comma separated list of machine hostname or IP would be installed Alert server, it
 # must be a subset of configuration `ips`.
-# Example for hostnames: ips="ds3", Example for IP: ips="192.168.8.3"
+# Example for hostname: alertServer="ds3", Example for IP: alertServer="192.168.8.3"
 alertServer="ds3"
 
 # A comma separated list of machine hostname or IP would be installed API server, it
 # must be a subset of configuration `ips`.
-# Example for hostnames: ips="ds1", Example for IP: ips="192.168.8.1"
+# Example for hostname: apiServers="ds1", Example for IP: apiServers="192.168.8.1"
 apiServers="ds1"
 
-# The directory to install DolphinScheduler for all machine we config above. It will automatically created by `install.sh` script if not exists.
-# **DO NOT** set this configuration same as the current path (pwd)
+# The directory to install DolphinScheduler for all machine we config above. It will automatically be created by `install.sh` script if not exists.
+# Do not set this configuration same as the current path (pwd)
```

Comment:
I think it's not necessary to highlight，so that keeping context consistent

Reply:
I thinks we should, but I do not have strong opinion here. maybe we should hear other opinion. cc: @CalvinKirs @ruanwenjun

Reply:
I have no preference for this. Can the highlight be rendered like Markdown? 
If it can, I think keep `**DO NOT**` is good, but if not, I think remove `**` is good.

Reply:
It can not render originally, need some change in our website, maybe we should remove it, cc @tracehh